lib/parsers/moov_parser.rb

```diff
@@ -163,5 +163,5 @@ def parse_sample_description_atom(decoder, atom_tree)
     end
   end
 
-  FormatParser.register_parser new, natures: :video, formats: FTYP_MAP.values, priority: 1
```

Comment:
Curious why this needs to be changed; I thought parsers should be listed in the order of likelihood of a random file matching it 🤔

Reply:
@linkyndy  there are currently 2 ways of matching a file to a certain parser, either you pass the format itself (e.g. "heic") in the format_parser as input alongside the file itself and then format_parser thanks to the **likely_match** method knows which parser to use, or you leave it up to the format_parser itself but priorities take place then. A HEIC file, due to its nature, could be associated (wrongly) to a .moov file and therefore if priorities take place then the moov parser would be used, which we don't want to. So I wanted to make space for the heif_parser to be in terms of priorities before the moov_parser, by at the same time making sure that nothing is broken on the moov_parser side.

Reply:
Ah, interesting! Thanks for explaining.